Resources/Prototypes/_Backmen/Entities/Structures/Machines/holopad.yml

```
@@ -0,0 +1,49 @@
+  suffix: Centcomm, Admiral
+  components:
+  - type: Label
+    currentLabel: holopad-centomm-admiral
+
+- type: entity
+  parent: HolopadCentCommBase
+  id: HolopadCentCommEvac
+  suffix: Centcomm, Evac room
+  components:
+  - type: Label
+    currentLabel: holopad-centomm-evac
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)